tests/repeatable_move/common.lua

```diff
@@ -11,7 +11,7 @@ function M.run_builtin_find_test(file, spec)
   assert.are.same(1, vim.fn.filereadable(file), string.format('File "%s" not readable', file))
 
   -- load reference file
-  vim.cmd(string.format("edit %s", file))
+  pcall(vim.cmd.edit, file)
```

Comment:
Why `pcall` instead of `vim.cmd.edit(file)`? Won't it silence the error when loading fails?

Reply:
I'm not sure, I just added it to be overkill; I didn't think that the `vim.cmd.edit` form of the function would handle the error gracefully if `vim.cmd('edit..` didn't

Reply:
But silencing the error is worse than failing, don't you think?

Reply:
True, so then maybe we need another different solution. But right now I think there are errors unrelated to the actual tests (caused by this function, *I think*) which is why some checks pass randomly and other times fail. I don't understand the test code well enough, but could it be the case that silencing this error will still allow the tests to run successfully? Seems like due to the nature of the error it just can't create a file because it already exists, so if we silence that we can just continue normally?

Reply:
It's not creating a file, rather it is reading a file. The tests read sample files and try multiple keystrokes to see if they have desirable behaviour. For example, it tests if `dia`, `viad`, `cia` results in the same buffer. Note that there were issues where it fails with certain combination that I don't really use but some people found it annoying.

So, if tests failed randomly it could be that we didn't wait for the keystrokes to finish (I thought they were synchronous but maybe not).